pages.es/common/ab.md

```diff
@@ -0,0 +1,24 @@
+# ab
+
+> Herramienta comparativa del servidor Apache HTTP.
+> Más información: <https://httpd.apache.org/docs/current/programs/ab.html>.
+
+- Ejecuta 100 solicitudes HTTP GET a una URL dada:
+
+`ab -n {{100}} {{url}}`
+
+- Ejecuta 100 solicitudes HTTP GET, en lotes simultáneos de a 10, a una URL:
+
+`ab -n {{100}} -c {{10}} {{url}}`
+
+- Ejectura 100 solicitudes HTTP POST a una URL, utilizando la carga JSON de un archivo:
```

Comment:
This verb is different than above two—is this a typo, are those, or are they all intentional?
```suggestion
- Ejecuta 100 solicitudes HTTP POST a una URL, utilizando la carga JSON de un archivo:
```